### DISCUSSION TITLE
tutorial-extras/ST7735

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#21] [comment]
# tutorial-extras/ST7735

简介 本章介绍使用Air001开发板驱动ST7735。 ST7735是一款具有SPI接口的彩色TFT液晶显示驱动芯片。 硬件准备 按☁️ Air001开发板入门 (/tutorial-advanced/Air001_start.html)，将Air001和`DAPLink调试器使用排针排母连接。; 将ST7735模块与Air001开发板，按如下表格进行...

https://arduino.luatos.com/tutorial-extras/ST7735.html

**Comment by ttyyzz34** [comment]
可以用ST7789吗？我看这个库都有，我想用它来驱动ST7789的屏幕。
！
